dd4t-core/src/main/java/org/dd4t/contentmodel/impl/ComponentPresentationImpl.java

```diff
@@ -34,7 +34,7 @@
  *
  * @author bjornl
  */
-public class ComponentPresentationImpl implements ComponentPresentation {
+public class ComponentPresentationImpl extends AbstractModel implements ComponentPresentation {

```

Comment:
This will break backward compatibility in case the eclId is not present in the Json